### PR TITLE
Rtrieu/docs 5550 update mobile vitals

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do?
- Updates/adds screenshots to reflect the new Mobile Vitals metrics and UI experience for Browser, Android, iOS, Flutter, and React native platforms

### Motivation
[DOCS-5550](https://datadoghq.atlassian.net/browse/DOCS-5550)

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-5550]: https://datadoghq.atlassian.net/browse/DOCS-5550?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ further_reading:
 
 RUM view events collect extensive performance metrics for every page view. Monitor your application's page views and explore performance metrics in dashboards and the RUM Explorer.
 
-{{< img src="real_user_monitoring/browser/waterfall.png" alt="A waterfall graph on the Performance tab of a RUM view in the RUM Explorer" style="width:100%;" >}}
+{{< img src="real_user_monitoring/browser/waterfall-3.png" alt="A waterfall graph on the Performance tab of a RUM view in the RUM Explorer" style="width:100%;" >}}
```
Comment (timmld) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:29]: there's a weird blank border on the right and top of the image of the preview

Screenshot 2023-06-21 at 12 12 05

File: content/en/real_user_monitoring/android/mobile_vitals.md [hunk 2]
```diff
@@ -17,13 +17,15 @@ Real User Monitoring offers Mobile Vitals, a set of metrics inspired by [Android
...
+{{< img src="real_user_monitoring/android/android_mobile_vitals-2.png" alt="Mobile Vitals in the RUM Explorer" style="width:90%;">}}
 
 To access the RUM mobile app performance dashboard, scroll down to the Monitor Performance Metrics section from the application summary page and click **Performance Overview**.
 
 {{< img src="real_user_monitoring/android/mobile-performance-dashboard-3.png" alt="Mobile Vitals in the Performance Tab" style="width:90%;">}}
 
-Understand your application's overall health and performance with the line graphs displaying metrics across various application versions. To filter on application version or see specific sessions and views, click on a graph. 
+Understand your application's overall health and performance with the line graphs displaying metrics across various application versions. To filter on application version or see specific sessions and views, click on a graph.
+
+{{< img src="real_user_monitoring/android/rum_explorer_mobile_vitals.png" alt="Event Timings and Mobile Vitals in the RUM Explorer" style="width:90%;">}}
```
Comment (timmld) [inline content/en/real_user_monitoring/android/mobile_vitals.md:28]: can we take the screenshot when the popover is open after clicking on a metric?

File: content/en/real_user_monitoring/android/mobile_vitals.md [hunk 3]
```diff
@@ -17,13 +17,15 @@ Real User Monitoring offers Mobile Vitals, a set of metrics inspired by [Android
...
 
 Mobile Vitals appear on your application's **Performance Overview** page when you navigate to **UX Monitoring > Performance Summary** and click your application. From the mobile performance dashboard for your application, click on a graph in **Mobile Vitals** to filter by version or examine filtered sessions.
```
Comment (timmld) [inline content/en/real_user_monitoring/android/mobile_vitals.md:30]: can we remove this paragraph? it's a repetition of what we have above

File: content/en/real_user_monitoring/android/mobile_vitals.md [hunk 4]
```diff
@@ -17,13 +17,15 @@ Real User Monitoring offers Mobile Vitals, a set of metrics inspired by [Android
 
 You can view Mobile Vitals for your application by navigating to  **UX Monitoring > Performance Summary** and selecting your application.
 
-{{< img src="real_user_monitoring/android/android_mobile_vitals-1.png" alt="Mobile Vitals in the RUM Explorer" style="width:90%;">}}
+{{< img src="real_user_monitoring/android/android_mobile_vitals-2.png" alt="Mobile Vitals in the RUM Explorer" style="width:90%;">}}
```
Comment (timmld) [inline content/en/real_user_monitoring/android/mobile_vitals.md:20]: shouldn't this be more a screenshot like this one? just like on the iOS page, to match the text

Screenshot 2023-06-21 at 12 22 50

File: content/en/real_user_monitoring/ios/mobile_vitals.md [hunk 5]
```diff
@@ -26,7 +26,7 @@ To access the RUM mobile app performance dashboard, scroll down to the Monitor P
 
 Understand your application's overall health and performance with the line graphs displaying metrics across various application versions. To filter on application version or see specific sessions and views, click on a graph. 
 
-{{< img src="real_user_monitoring/ios/rum_explorer_mobile_vitals.png" alt="Event Timings and Mobile Vitals in the RUM Explorer" style="width:90%;">}}
+{{< img src="real_user_monitoring/ios/rum_explorer_mobile_vitals-2.png" alt="Event Timings and Mobile Vitals in the RUM Explorer" style="width:90%;">}}
```
Comment (timmld) [inline content/en/real_user_monitoring/ios/mobile_vitals.md:29]: same comment, can we see the metric popover open?

File: content/en/real_user_monitoring/reactnative/mobile_vitals.md [hunk 6]
```diff
@@ -28,6 +28,8 @@ To access the RUM mobile app performance dashboard, scroll down to the Monitor P
 
 Understand your application's overall health and performance with the line graphs displaying metrics across various application versions. To filter on application version or see specific sessions and views, click on a graph.
 
+{{< img src="real_user_monitoring/react_native/rum_explorer_mobile_vitals-1.png" alt="Event Timings and Mobile Vitals in the RUM Explorer" style="width:90%;">}}
```
Comment (timmld) [inline content/en/real_user_monitoring/reactnative/mobile_vitals.md:31]: same comment, can we see the metric popover open?

File: content/en/real_user_monitoring/reactnative/mobile_vitals.md [hunk 7]
```diff
@@ -28,6 +28,8 @@ To access the RUM mobile app performance dashboard, scroll down to the Monitor P
```
Comment (timmld) [inline content/en/real_user_monitoring/reactnative/mobile_vitals.md:28]: the first image is that of the flutter summary page (widget build time is a flutter vital), can we change it to the react native page?

File: content/en/real_user_monitoring/reactnative/mobile_vitals.md [hunk 8]
```diff
@@ -28,6 +28,8 @@ To access the RUM mobile app performance dashboard, scroll down to the Monitor P
 
 Understand your application's overall health and performance with the line graphs displaying metrics across various application versions. To filter on application version or see specific sessions and views, click on a graph.
 
+{{< img src="real_user_monitoring/react_native/rum_explorer_mobile_vitals-1.png" alt="Event Timings and Mobile Vitals in the RUM Explorer" style="width:90%;">}}
+
 You can also select a view in the RUM Explorer and observe recommended benchmark ranges that directly correlate to your application's user experience in the session. Click on a metric such as **Refresh Rate Average** and click **Search Views With Poor Performance** to apply a filter in your search query and examine additional views.
 
 ## Metrics
```
Comment (timmld) [inline content/en/real_user_monitoring/reactnative/mobile_vitals.md:35]: can we add the `JS Rrefresh Rate` (or `JS Slow Renders`) in this list? @louiszawadzki or @fuzzybinary can advise if needed

File: content/en/real_user_monitoring/flutter/mobile_vitals.md [hunk 9]
```diff
@@ -18,17 +18,25 @@ further_reading:
...
 
 To access the RUM mobile app performance dashboard, scroll down to the Monitor Performance Metrics section from the application summary page and click **Performance Overview**.
 
 {{< img src="real_user_monitoring/flutter/flutter-performance-overview.png" alt="Mobile performance overview dashboard" style="width:90%;">}}
 
-Understand your application's overall health and performance with the line graphs displaying metrics across various application versions. To filter on application version or see specific sessions and views, click on a graph. 
+Understand your application's overall health and performance with the line graphs displaying metrics across various application versions. To filter on application version or see specific sessions and views, click on a graph.
 
-{{< img src="real_user_monitoring/ios/rum_explorer_mobile_vitals.png" alt="Event Timings and Mobile Vitals in the RUM Explorer" style="width:90%;">}}
+When viewing performance details by the **View** event type, the following event timing and mobile vitals information is available:
```
Comment (timmld) [inline content/en/real_user_monitoring/flutter/mobile_vitals.md:31]: i'd say let's keep it standard across all docs pages and not have this list here, but rather directly in the table below

File: content/en/real_user_monitoring/flutter/mobile_vitals.md [hunk 10]
```diff
@@ -18,17 +18,25 @@ further_reading:
...
+
+- Refresh Rate Average
+- CPU Tickets Per Second
+- Memory Average
+- View Flutter Build TIme Average
+- Raster Time Average
+
+{{< img src="real_user_monitoring/flutter/rum_explorer_mobile_vitals-2.png" alt="Event Timings and Mobile Vitals in the RUM Explorer" style="width:90%;">}}
```
Comment (timmld) [inline content/en/real_user_monitoring/flutter/mobile_vitals.md:39]: same comment, can we see the metric popover open?

File: content/en/real_user_monitoring/flutter/mobile_vitals.md [hunk 11]
```diff
@@ -18,17 +18,25 @@ further_reading:
...
 
 You can also select a view in the RUM Explorer and observe recommended benchmark ranges that directly correlate to your application's user experience in the session. Click on a metric such as **Refresh Rate Average** and click **Search Views With Poor Performance** to apply a filter in your search query and examine additional views.
```
Comment (timmld) [inline content/en/real_user_monitoring/flutter/mobile_vitals.md:42]: for standardization, can we remove the section below this? and copy paste the same table we have for android, ios, or react native? and make sure we add widget build time and raster time in it (@fuzzybinary can advise if needed)

**Review by timmld — CHANGES_REQUESTED** [review]
left a few suggestions, many of them are repetitive bc they're the same for all 4 android, ios, react native, and flutter pages

File: content/en/real_user_monitoring/reactnative/mobile_vitals.md [hunk 12]
```diff
@@ -28,6 +28,8 @@ To access the RUM mobile app performance dashboard, scroll down to the Monitor P
 
 Understand your application's overall health and performance with the line graphs displaying metrics across various application versions. To filter on application version or see specific sessions and views, click on a graph.
```
Comment (louiszawadzki) [inline content/en/real_user_monitoring/reactnative/mobile_vitals.md:29]: Can we add a section similar to the one we have for Flutter to advertise for the JS refresh rate?

Something like so:

```suggestion
Understand your application's overall health and performance with the line graphs displaying metrics across various application versions. To filter on application version or see specific sessions and views, click on a graph.

When viewing performance details by the **View** event type, the following event timing and mobile vitals information is available:

- UI Refresh Rate Average
- JS Refresh Rate Average
- CPU Tickets Per Second
- Memory Average
```

File: content/en/real_user_monitoring/flutter/mobile_vitals.md [hunk 13]
```diff
@@ -18,17 +18,25 @@ further_reading:
...
-Understand your application's overall health and performance with the line graphs displaying metrics across various application versions. To filter on application version or see specific sessions and views, click on a graph. 
+Understand your application's overall health and performance with the line graphs displaying metrics across various application versions. To filter on application version or see specific sessions and views, click on a graph.
 
-{{< img src="real_user_monitoring/ios/rum_explorer_mobile_vitals.png" alt="Event Timings and Mobile Vitals in the RUM Explorer" style="width:90%;">}}
+When viewing performance details by the **View** event type, the following event timing and mobile vitals information is available:
+
+- Refresh Rate Average
+- CPU Tickets Per Second
+- Memory Average
+- View Flutter Build TIme Average
```
Comment (fuzzybinary) [inline content/en/real_user_monitoring/flutter/mobile_vitals.md:36]: I think this changed to be `Build Time Average` in latest code but @siobhanpmahoney to confirm

File: content/en/real_user_monitoring/android/mobile_vitals.md (hunk 2 above)
Comment (rtrieu) [reply content/en/real_user_monitoring/android/mobile_vitals.md:28]: @timmld Updated the screenshot - is this what you meant?

File: content/en/real_user_monitoring/reactnative/mobile_vitals.md (hunk 12 above)
Comment (rtrieu) [reply content/en/real_user_monitoring/reactnative/mobile_vitals.md:29]: Hi @louiszawadzki, based on @timmld's feedback, I will actually remove this list in favor of a table similar to the one here: https://docs.datadoghq.com/real_user_monitoring/android/mobile_vitals/#metrics. I could use your help in defining these metrics!

File: content/en/real_user_monitoring/flutter/mobile_vitals.md (hunk 11 above)
Comment (rtrieu) [reply content/en/real_user_monitoring/flutter/mobile_vitals.md:42]: @fuzzybinary would you mind filling in the blanks here?

File: content/en/real_user_monitoring/reactnative/mobile_vitals.md (hunk 8 above)
Comment (rtrieu) [reply content/en/real_user_monitoring/reactnative/mobile_vitals.md:35]: @louiszawadzki @fuzzybinary can you please help fill in the blanks here?

File: content/en/real_user_monitoring/reactnative/mobile_vitals.md (hunk 12 above)
Comment (louiszawadzki) [reply content/en/real_user_monitoring/reactnative/mobile_vitals.md:29]: Awesome! For CPU ticks and memory usage you can reuse the definition from your link.

We also have **Slow renders** which are linked to Refresh Rates but less important, so I would suggest to update it and put it after the refresh rates whose definition could be:

**Refresh Rate Average**
To ensure a smooth, [jank-free](http://jankfree.org/) user experience, your application should render frames in under 60Hz.

RUM tracks the application’s [main thread display refresh rate](https://reactnative.dev/docs/performance#what-you-need-to-know-about-frames) using @view.refresh_rate_average and @view.refresh_rate_min view attributes.

Note: Refresh rates are normalized on a range of zero to 60fps. For example, if your application runs at 100fps on a device capable of rendering 120fps, Datadog reports 50fps in Mobile Vitals.

**JS Refresh Rate Average**
To ensure a smooth, [jank-free](http://jankfree.org/) user experience, your application should render frames in under 60Hz.

RUM tracks the application’s [javascript thread display refresh rate](https://reactnative.dev/docs/performance#what-you-need-to-know-about-frames) using @view.js_refresh_rate.average, @view.js_refresh_rate.min, @view.js_refresh_rate.max view attributes.

Note: Refresh rates are normalized on a range of zero to 60fps. For example, if your application runs at 100fps on a device capable of rendering 120fps, Datadog reports 50fps in Mobile Vitals.

**Slow renders**
To ensure a smooth, [jank-free](http://jankfree.org/) user experience, your application should render frames in under 60Hz.

With slow rendering, you can monitor which views have an average frame rate under 55fps.

Note: Refresh rates are normalized on a range of zero to 60fps. For example, if your application runs at 100fps on a device capable of rendering 120fps, Datadog reports 50fps in Mobile Vitals.

File: content/en/real_user_monitoring/flutter/mobile_vitals.md (hunk 11 above)
Comment (fuzzybinary) [reply content/en/real_user_monitoring/flutter/mobile_vitals.md:34]: ** Build Time Average **
This is the duration of time taken build the frame on the UI thread. To ensure a smooth animations, this should not exceed 16ms for 60 FPS, and 8ms for 120 FPS.  High values here mean you need to look into optimizing your `build` methods for this view. See [Control Build Cost](https://docs.flutter.dev/perf/best-practices#control-build-cost) in the Flutter documentation

** Raster Time Average **
This is the duration of time taken to rasterize the frame on the raster thread. To ensure a smooth animations, this should not exceed 16ms for 60 FPS, and 8ms for 120 FPS.  High values here may mean your view is complex to render. See [Identifying Problems in the GPU Graph](https://docs.flutter.dev/perf/ui-performance#identifying-problems-in-the-gpu-graph) in the Flutter documentation.

**Review by estherk15 — COMMENTED** [review]
@rtrieu, this looks good, ping me for a re-review if you end up adding the table in this PR.

File: content/en/real_user_monitoring/android/mobile_vitals.md (hunk 2 above)
Comment (timmld) [reply content/en/real_user_monitoring/android/mobile_vitals.md]: exactly, thanks!

**Comment by timmld** [comment]
@estherk15 i see Rosa's on PTO, so i made the changes for her. would be able to have a look and (hopefully) merge today? that'd be great, so we can announce the feature right away

**Review by timmld — APPROVED** [review]

**Review by louiszawadzki — APPROVED** [review]

File: content/en/real_user_monitoring/reactnative/mobile_vitals.md [hunk 14]
```diff
@@ -20,27 +20,35 @@ Real User Monitoring offers Mobile Vitals, a set of metrics that can help comput
...
 | Memory utilization             | High memory usage can lead to [out-of-memory crashes][9], which causes a poor user experience. <br /><br />  RUM tracks the amount of physical memory used by your application in bytes for each view, over the course of a session. The recommended range is <200MB for good and <400MB for moderate. <br /><br />  You can see the top views with the most memory consumption on average over a selected time period under **Mobile Vitals** in your application's Overview page.                                                                                                                                                                                                                                                                                            |
+| UI refresh rate average | |
```
Comment (estherk15) [inline content/en/real_user_monitoring/reactnative/mobile_vitals.md:48]: @timmld Are these supposed to be filled in? Or would be be better to remove them until they can be filled in later?

Comment (timmld) [reply content/en/real_user_monitoring/reactnative/mobile_vitals.md:48]: sorry, that can be removed - good catch!

**Review by estherk15 — APPROVED** [review]

**Pull request merged**